libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -2654,6 +2622,40 @@ async def ainvoke(
         return await self._acall_with_config(self._ainvoke, input, config, **kwargs)
 
 
+class RunnableEach(RunnableEachBase[Input, Output]):
```

Comment:
Could you add a doc-string explaining why there's a Base vs. no Base?

Reply:
Done